.github/workflows/stale-issues.yml

```diff
@@ -0,0 +1,30 @@
+name: Mark and Close Stale Issues
+
+on:
+  schedule:
+  - cron: 0 */12 * * * # Every 12 hours
+
+jobs:
+  staleIssues:
+    # Only run this on repositories in the 'spinnaker' org, not on forks.
+    if: startsWith(github.repository, 'spinnaker/')
+    runs-on: ubuntu-latest
+    steps:
+    - name: stale
+      uses: gatsbyjs/stale@master
```

Comment:
```suggestion
      # github release 0.1.0
      uses: gatsbyjs/stale@0dde19eb39fa2003923986adc3608c3e9178ebf3
```

Reply:
I recommend tracking a specific release, and also using the exact commit sha (since tags can be deleted and re-created)